### PR TITLE
Design token components MVP

### PR DESCRIPTION
**Pull request by sapegin**
image

### CODE REVIEW

File: src/styleguide/tokens/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	height: ${props => (props.size === 0 ? '1px' : props.size)};
+	background-color: ${props => (props.size === 0 ? '#efefef' : '#ccc')};
+`;
+
+const SpacingSwatchLabel = styled(Box)`
+	min-width: 5rem;
+`;
+
+const SpacingSwatchSubValue = styled(Box)`
+	color: #767676;
```
Comment (okonet) [inline src/styleguide/tokens/index.js:102]: Perhaps just use `css` prop instead adding new components?

**Review by okonet — APPROVED** [review]

**Comment by okonet** [comment]
LGTM and I can check styling later. Id remove margin between color swatches

**Comment by sapegin** [comment]
I've changed the API a bit to make spacing more flexible.

**Comment by sapegin** [comment]
What do you mean by "more uniform"? `items` is a list, `children` is a render prop to render each item.

**Comment by okonet** [comment]
That’s what I mean! If we have children prop why do we need to pass items?

**Comment by sapegin** [comment]
In this case we can just use stack-styled directly.

Artem Sapegin, sapegin.me

**Comment by okonet** [comment]
True. Maybe it’s the naming? I think i would expect `itemRenderer` with `items`. `children` makes it feel wrong for me. WYT?

**Comment by sapegin** [comment]
I'm used to children as a function pattern, but `items` / `renderItem` looks good too, connection between two would be more clear.

**Review by okonet — APPROVED** [review]

**Pull request merged**